lib/acme/client.rb

```diff
@@ -127,8 +128,16 @@ def finalize(url:, csr:)
     Acme::Client::Resources::Order.new(self, **arguments)
   end
 
-  def certificate(url:)
+  def certificate(url:, fetch_alternative_chains: false)
     response = download(url, format: :pem)
+    if fetch_alternative_chains
+      @alternative_certificates = []
+      alt_chains_urls = fetch_alternative_links(response)
+      alt_chains_urls.each do |alt_chains_url|
+        response = download(alt_chains_url, format: :pem)
```

Comment:
This re-assignment of `response` each time through the loop makes it impossible to get the original certificate response without invoking the method twice (once with `fetch_alternative_chains: false` and once with `fetch_alternative_chains: true`)

It also makes the value returned by this method unpredictable depending on the order of alternative certificates within the `Links` header.

As someone from the community using this library heavily, my preference would be to have a new method added (eg. `alternative_certificates`) and to have this existing method only modified to persist the `Links` in an instance variable, perhaps. This keeps the API compatible and separates the concerns a bit more.

Reply:
> persist the Links in an instance variable

Well, it's definitely better that way 👍  I'll try to change it to do so.